browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -216,7 +220,7 @@ public void captureResult(@NonNull FragmentActivity activity) {
         Uri deepLinkUrl = intent.getData();
         if (deepLinkUrl != null) {
             BrowserSwitchResult result =
-                new BrowserSwitchResult(BrowserSwitchStatus.SUCCESS, request, deepLinkUrl);
+                    new BrowserSwitchResult(BrowserSwitchStatus.SUCCESS, request, deepLinkUrl);
```

Comment:
Is this the indentation level we want?

Reply:
Yeah it's the one enforced by the linter. It defaults to a double indent for continuation lines. We don't have a hard convention in place yet though might be worth changing.